test/lib/operation.go

```diff
@@ -121,6 +122,38 @@ func (client *Client) sendFakeEventWithTracingToAddress(
 	return nil
 }
 
+// ConfigMapExists will check if a configmap exists in a specific namespace
+func (client *Client) ConfigMapExist(namespace string, names ...string) error {
+	if names != nil {
```

Comment:
Do we need this check?

Reply:
I'll delete it, as `ConfigMapEqual` also checks the existence.

Reply:
What I meant is actually if we need the `if names != nil` check.. Sorry for the confusion!

Reply:
Do I still need to keep this function? I felt that as `ConfigMapEqual` also checks the existence, it might be duplicated to keep it.

Reply:
I see, this can be deleted. But it would be better to indicate this kind of error in `ConfigMapEqual`, e.g. replace `err` with `fmt.Errorf("failed to get the copied configmap")`